scripts/fixtures_qa/one_type.json

```diff
@@ -0,0 +1,119 @@
+[
+        {
+            "mimetype": 50,
+            "type": {
+                "name": "Generic",
+                "major_version": 1, 
+                "minor_version": 0, 
+                "patch_version": 0
+            },
```

Comment:
Instead of the `type` field, in this file can we add two fields called `schema_name` and `schema_version`? e.g.

```
"schema_name": "Trade",
"schema_version": "1.0.0"
```

Reply:
This will be more compatible with the upcoming version of PyEnsign.